CONTRIBUTING.md

```diff
@@ -4,18 +4,18 @@ PaperMC is happy you're willing to contribute to our projects. We are usually
 very lenient with all submitted PRs, but there are still some guidelines you
 can follow to make the approval process go more smoothly.
 
-## Use a Personal Fork and not an Organization
+## Branch Protection
 
 Paper will routinely modify your PR, whether it's a quick rebase or to take care
 of any minor nitpicks we might have. Often, it's better for us to solve these
 problems for you than make you go back and forth trying to fix them yourself.
 
-Unfortunately, if you use an organization for your PR, it prevents Paper from
-modifying it. This requires us to manually merge your PR, resulting in us
-closing the PR instead of marking it as merged.
+Unfortunately, if you use an organization or the master branch for your PR, 
```

Comment:
We close PRs from master as it complicates the dev workflow, not related to the write issue with organization forks.

Reply:
Been a while since i took a look at this pr. what exactly was the deal with master branch again? iirc you guys have to change something in gh settings to be a able to push the a forks master? could you or anyone else please elaborate on that.

Reply:
I think github used to create issues with it, no idea if that's still an issue or not; but, a common issue is that it often creates issues for people who are unfamiliar with git when stuff diverges, things like managing more than 2 PRs, etc

Reply:
does something like "you also shouldn't create PRs from the master branch as this is bad practice and may cause problems in the dev workflow" (not final just a rough idea)